File: API/Features/Reservations/PickupPoints/ViewModels/PickupPointListVM.cs
namespace API.Features.Reservations.PickupPoints {

    public class PickupPointListVM {

        public int Id { get; set; }
        public string Description { get; set; }
        public string LinkTwistAlias { get; set; }
        public PickupPointListCoachRouteVM CoachRoute { get; set; }
        public PickupPointListPortVM Port { get; set; }
        public string ExactPoint { get; set; }
        public string Time { get; set; }
        public string Remarks { get; set; }
        public string PutAt { get; set; }
        public bool IsActive { get; set; }

    }

}

File: FrontEnd/src/app/features/reservations/pickupPoints/classes/view-models/pickupPoint-list-vm.ts
import { PickupPointListCoachRouteVM } from './pickupPoint-list-coachRoute-vm'
import { PickupPointListPortVM } from './pickupPoint-list-port-vm'

export interface PickupPointListVM {

    id: number
    description: string
    linkTwistAlias: string
    coachRoute: PickupPointListCoachRouteVM
    port: PickupPointListPortVM
    exactPoint: string
    time: string
    remarks: string
    putAt: string
    isTemp: boolean
    isActive: boolean

}
